app/src/common/shared/org/mozilla/vrbrowser/browser/engine/SessionUtils.java

```diff
@@ -25,12 +26,38 @@ public static boolean isLocalizedContent(@Nullable String url) {
         return url != null && (url.startsWith("about:") || url.startsWith("data:"));
     }
 
+    private static void checkPrefsFile(Context aContext, @NonNull String path) {
+        int lastVersionCode = SettingsStore.getInstance(aContext).getPrefsLastResetVersionCode();
+        if (BuildConfig.VERSION_CODE > lastVersionCode) {
```

Comment:
We should use != instead of for downgrades

Reply:
You have to uninstall first. You can’t install an older version number over a newer one.

Reply:
ok, it can only be done for debuggable builds which is not the case of Taskcluster.

Reply:
Verified debug taskcluster builds have the same behavior as release builds.